src/main/webapp/trip/tripDetails.html

```diff
@@ -2,46 +2,126 @@
 
         <!-- Favicon -->
-        <link rel="icon" href="images/favicon.ico">
+        <link rel="icon" href="../images/favicon.ico">
     </head>
 
     <body>
         <div class="container">
-            <h1 class="home-title">Trip name</h1>
-            <h3 class="home-subtitle">Add participants</h3>
-            <form action="/NewPeople" method="POST">
-                <div class="inputBox">
-                    <input name="text-input-participants" type="text" placeholder="Patrick" required="required">
-                    <input type="submit" value="add">
+            <h1 class="trip-title">Trip name</h1>
+            <div class="enter-btn-div">
+
+                <!-- Participants -->
+                <h3 class="trip-item">💚 Add participants</h3>
+
+                <!-- <div class="select-box">
+                    <div class="options-container">
+                      <div class="option">
+                        <input type="radio" class="radio" id="emma" name="participant"/>
+                        <label for="emma">Emma</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="jack" name="participant" />
+                        <label for="jack">Jack</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="jess" name="participant" />
+                        <label for="jess">Jess</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="mauricio" name="participant" />
+                        <label for="mauricio">Mauricio</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="nikki" name="participant" />
+                        <label for="nikki">Nikki</label>
+                      </div>
+                      <div class="option">
+                        <input type="radio" class="radio" id="shaheen" name="participant" />
+                        <label for="shaheen">Shaheen</label>
+                      </div>
+            
+                      <div class="option">
+                        <input type="radio" class="radio" id="music" name="category" />
+                        <label for="music">Yifan</label>
+                      </div>
+                    </div>
+        
+                    <div class="selected" onclick="select()">
+                      Select a Participant
+                    </div>
+                  </div> -->
+
+                <h3 class="trip-item-or">or</h3>
+                <form action="/NewPeople" method="POST">
+                    <div class="inputBox">
+                        <input name="text-input-participants" type="text" placeholder="e.g. Patrick">
```

Comment:
Could you explain what the user input will look like in the group chat. I'm a bit confused with how it'll use the drop down. Currently our method is expecting a comma separated list.

Reply:
They are strings

Reply:
Good to know